addons/project/static/src/web/activity/activity_menu_patch.js

```diff
@@ -0,0 +1,27 @@
+/* @odoo-module */
+
+import { ActivityMenu } from "@mail/web/activity/activity_menu";
+import { patch } from "@web/core/utils/patch";
+
+patch(ActivityMenu.prototype, "task", {
+
+    openActivityGroup(group, filter = "all") {
+        // fetch the data from the button otherwise fetch the ones from the parent (.o_ActivityMenuView_activityGroup).
+        const context = {};
+        if (group.model === "project.task") {
```

Comment:
```suggestion
        if (group.model !== "project.task") {
            return this._super(...arguments);
        }
        const context = {};
        ...
```
To gain an indent :slightly_smiling_face: